src/test/java/edu/unc/lib/boxc/migration/cdm/test/RedirectMappingHelper.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.migration.cdm.test;
+
+import edu.unc.lib.boxc.migration.cdm.exceptions.MigrationException;
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import edu.unc.lib.boxc.migration.cdm.options.SipGenerationOptions;
+import edu.unc.lib.boxc.migration.cdm.services.CdmIndexService;
+import org.apache.commons.io.FileUtils;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+/**
+ * @author snluong
+ */
+public class RedirectMappingHelper {
+    private static final String USERNAME = "migr_user";
+    public static final String REDIRECT_MAPPING_INDEX_FILENAME = "redirect_mapping_index.db";
+    private final MigrationProject project;
+
+    public RedirectMappingHelper(MigrationProject project) {
+        this.project = project;
+    }
+
+    public void createRedirectMappingsTableInDb() {
+        String connectionString = "jdbc:sqlite:" + getRedirectMappingIndexPath();
```

Comment:
for what its worth, it seems like you could use `RedirectMappingIndexService.openDbConnection` for initializing the database connection, which would make sure they work the same way, although it would only save you a few lines here

Reply:
yeah that's true. i'll change it after my meeting at 2

Reply:
upon further thought, i'll just keep it simple as is =)